models/sipnet/R/met2model.SIPNET.R

```diff
@@ -147,12 +147,15 @@ met2model.SIPNET <- function(in.path, in.prefix, outfolder, start_date, end_date
       }
       
       Rain <- ncdf4::ncvar_get(nc, "precipitation_flux")
-      pres <- ncdf4::ncvar_get(nc,'air_pressure') ## in pascal
+      
+      press <- ncdf4::ncvar_get(nc,'air_pressure') ## in pascal
```

Comment:
glad you added the additional "s"

Reply:
Collaborative coding is fixing other peoples typos one word at a time haha